packages/packages.csv

```diff
@@ -1,174 +1,174 @@
-Microsoft.Extensions.RuntimeEnvironment.Sources,noship
-Microsoft.Extensions.SecurityHelper.Sources,noship
-Microsoft.Extensions.StackTrace.Sources,noship
-Microsoft.Extensions.TypeNameHelper.Sources,noship
-Microsoft.Extensions.WebEncoders,ship
-Microsoft.Net.Http.Headers,ship
-Microsoft.Net.Http.Server,ship
-Microsoft.Net.WebSockets.Server,ship
-Microsoft.Owin.Security.Interop,ship
-Microsoft.VisualStudio.Web.BrowserLink.Loader,ship
-PageGenerator,noship
+Package,Category,OptimizedCache
+Microsoft.AspNetCore.Razor.Tools,ship,exclude
+Microsoft.VisualStudio.Web.CodeGeneration.Tools,ship,exclude
+Microsoft.Extensions.SecretManager.Tools,ship,exclude
+Microsoft.DotNet.Watcher.Tools,ship,exclude
+Microsoft.DotNet.Watcher.Core,ship,exclude
+Microsoft.AspNet.SignalR.Client.Cpp.v120.WinDesktop,noship,exclude
+Microsoft.AspNet.SignalR.Client.Cpp.v140.WinDesktop,noship,exclude
+Microsoft.AspNetCore.Antiforgery,ship,include
+Microsoft.AspNetCore.Authentication,ship,include
+Microsoft.AspNetCore.Authentication.Cookies,ship,include
+Microsoft.AspNetCore.Authentication.Facebook,ship,include
+Microsoft.AspNetCore.Authentication.Google,ship,include
+Microsoft.AspNetCore.Authentication.JwtBearer,ship,include
+Microsoft.AspNetCore.Authentication.MicrosoftAccount,ship,include
+Microsoft.AspNetCore.Authentication.OAuth,ship,include
+Microsoft.AspNetCore.Authentication.OpenIdConnect,ship,include
+Microsoft.AspNetCore.Authentication.Twitter,ship,include
+Microsoft.AspNetCore.Authorization,ship,include
+Microsoft.AspNetCore.Buffering,ship,include
+Microsoft.AspNetCore.CookiePolicy,ship,include
+Microsoft.AspNetCore.Cors,ship,include
+Microsoft.AspNetCore.Cryptography.Internal,ship,include
+Microsoft.AspNetCore.Cryptography.KeyDerivation,ship,include
+Microsoft.AspNetCore.DataProtection,ship,include
+Microsoft.AspNetCore.DataProtection.Abstractions,ship,include
+Microsoft.AspNetCore.DataProtection.Extensions,ship,include
+Microsoft.AspNetCore.DataProtection.Sources,noship,exclude
+Microsoft.AspNetCore.DataProtection.SystemWeb,ship,exclude
+Microsoft.AspNetCore.Diagnostics,ship,include
+Microsoft.AspNetCore.Diagnostics.Abstractions,ship,include
+Microsoft.AspNetCore.Diagnostics.Elm,ship,include
+Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore,ship,include
+Microsoft.AspNetCore.DiagnosticsViewPage.Sources,noship,exclude
+Microsoft.AspNetCore.Hosting,ship,include
+Microsoft.AspNetCore.Hosting.Abstractions,ship,include
+Microsoft.AspNetCore.Hosting.Server.Abstractions,ship,include
+Microsoft.AspNetCore.Hosting.WindowsServices,ship,exclude
+Microsoft.AspNetCore.Html.Abstractions,ship,include
+Microsoft.AspNetCore.Http,ship,include
+Microsoft.AspNetCore.Http.Abstractions,ship,include
+Microsoft.AspNetCore.Http.Extensions,ship,include
+Microsoft.AspNetCore.Http.Features,ship,include
+Microsoft.AspNetCore.HttpOverrides,ship,include
+Microsoft.AspNetCore.Identity,ship,include
+Microsoft.AspNet.Identity.AspNetCoreCompat,ship,exclude
```

Comment:
I'm not sure what implications that has, this is a preview package for getting identity 2.0 to interop with Identity core, so its not really a mainline scenario.